--- sections.ld	2025-08-23 00:10:34.484347738 +0200
+++ sections-ecrf.ld	2025-08-23 00:12:06.357523897 +0200
@@ -852,6 +852,12 @@
     *(.rodata_desc .rodata_desc.*)
     /* Should be the second. Custom app version info. */
     *(.rodata_custom_desc .rodata_custom_desc.*)
+
+    . = ALIGN(16);
+    __freertos_shell_cmd_start = ABSOLUTE(.);
+    KEEP(*(.FREERTOS_SHELL_CMD_SECTION))
+    __freertos_shell_cmd_end = ABSOLUTE(.);
+
     /**
      * Create an empty gap within this section. Thanks to this, the end of this
      * section will match .flah.rodata's begin address. Thus, both sections
